src/ck-core/ckarray.C

```diff
@@ -1370,6 +1370,27 @@ bool CkArrayBroadcaster::deliver(CkArrayMessage* bcast, ArrayElement* el, bool d
   }
 }
 
+bool CkArrayBroadcaster::deliverAlreadyDelivered(CkArrayMessage *bcast, ArrayElement *el, bool doFree)
```

Comment:
@epmikida: I added this method for delivering a message to an element after the posting is completed. It is similar to the previous version of `CkArrayBroadcaster::deliever`. I'm guessing this needs to be updated now?

Reply:
This looks fine. It may change slightly with new delivery code, but at least for now I believe it is accurate.